src/screens/TopicMemberScreen/index.js

```diff
@@ -154,7 +154,6 @@ const TopicMemberScreen = () => {
     cancelTokenRef.current = axios.CancelToken.source();
   };
 
-  if (isInitialLoading) return null;
   return (
     <SafeAreaProvider forceInset={{top: 'always'}} style={styles.parentContainer}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the conditional rendering based on `isInitialLoading` could potentially cause issues if the data required for rendering isn't available yet. If `isInitialLoading` was used to ensure that certain data was loaded before attempting to render components, removing this check might lead to errors or incomplete rendering. Please ensure that all necessary data is either available at render time or handled appropriately within the child components.
```diff
-  if (isInitialLoading) return null;
   return (
     <SafeAreaProvider forceInset={{top: 'always'}} style={styles.parentContainer}>
       <NavHeader
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->